src/bin/client-demo.rs

```diff
@@ -33,9 +33,14 @@ use std::thread::JoinHandle;
 use std::time::Duration;
 use std::time::Instant;
 
+pub struct NodeStats {
+    pub tps: f64,
+    pub tx: u64,
```

Comment:
can you please document these two fields?  I can mostly guess what they mean but I'd rather not guess :)

Reply:
Done..